code/modules/cargo/exports/food_and_drink.dm

```diff
@@ -0,0 +1,361 @@
+		/obj/item/reagent_containers/food/snacks/donut/chaos,
+		/obj/item/reagent_containers/food/snacks/donut/blumpkin,
+		/obj/item/reagent_containers/food/snacks/donut/jelly/blumpkin,
+		/obj/item/reagent_containers/food/snacks/donut/bungo,
+		/obj/item/reagent_containers/food/snacks/donut/jelly/bungo,
+		/obj/item/reagent_containers/food/snacks/donut/matcha,
+		/obj/item/reagent_containers/food/snacks/donut/jelly/matcha,
+		/obj/item/reagent_containers/food/snacks/donut/jelly/slimejelly, //Because honestly, if you're even making slime jelly, that's impressive
+		/obj/item/reagent_containers/food/snacks/muffin/booberry,
+		/obj/item/reagent_containers/food/snacks/chawanmushi,
+		/obj/item/reagent_containers/food/snacks/rofflewaffles,
+		/obj/item/reagent_containers/food/snacks/meatbun,
+		/obj/item/reagent_containers/food/snacks/pie/bearypie,
+		/obj/item/reagent_containers/food/snacks/pie/xemeatpie,
+		/obj/item/reagent_containers/food/snacks/pie/appletart,
+		/obj/item/reagent_containers/food/snacks/pie/mimetart,
+		/obj/item/reagent_containers/food/snacks/pie/cocolavatart,
+		/obj/item/reagent_containers/food/snacks/pie/blumpkinpie,
+		/obj/item/reagent_containers/food/snacks/pie/dulcedebatata,
+		/obj/item/reagent_containers/food/snacks/pie/baklava,
+		/obj/item/reagent_containers/food/snacks/pizza/sassysage,
+		/obj/item/reagent_containers/food/snacks/salad/aesirsalad,
+		/obj/item/reagent_containers/food/snacks/soup/blood,
+		/obj/item/reagent_containers/food/snacks/soup/wingfangchu,
+		/obj/item/reagent_containers/food/snacks/soup/mystery,
+		/obj/item/reagent_containers/food/snacks/soup/coldchili,
+		/obj/item/reagent_containers/food/snacks/soup/clownchili,
+		/obj/item/reagent_containers/food/snacks/soup/stew,
+		/obj/item/reagent_containers/food/snacks/soup/bisque,
+		/obj/item/reagent_containers/food/snacks/soup/electron,
+		/obj/item/reagent_containers/food/snacks/soup/bungocurry)
+	include_subtypes = TRUE
+
+//Tier 5:Legendary Foods.
+//If you can make one of these in an hour long round, without any help, then you'll surely be awarded.
+//Each one sold should be basically a payday and a half.
+/datum/export/food/legendary
+	cost = 500
+	message = "of breathtaking food"
+	export_types = list(
+		/obj/item/reagent_containers/food/snacks/burger/roburgerbig,
+		/obj/item/reagent_containers/food/snacks/powercrepe,
+		/obj/item/reagent_containers/food/snacks/stuffedlegion,
+		/obj/item/reagent_containers/food/snacks/soup/clownstears)
+	include_subtypes = TRUE
+
+//Tier 0: Illegal Foods.
+//Yeah... centcom's not gonna buy these. Whatsoever.
+//But, if you know a guy, who knows a guy, he might just take it off your hands for ya, for a pretty good price, too.
+/datum/export/food/illegal
+	cost = 500
+	message = "of quote-unquote 'food'"
+	export_category = EXPORT_CONTRABAND
+	export_types = list(
+		/obj/item/reagent_containers/food/snacks/store/cake/birthday/energy,
+		/obj/item/reagent_containers/food/snacks/pizza/arnold)
```

Comment:
you do know that you can make arnold pizza without using any antag items, right?

also, the energy cake should sell for WAY more, you're pretty much selling an esword here (remember that a mere 2 TC can buy a syndicate briefcase of cash!)

Reply:
I know it's not an antag item, yes. All it takes is a hacked autolathe to make one.
That said, if you're actually cooking this for other people in the round, you will almost CERTAINLY do more harm than good, and based on the amount of cooperation required to get a reliable source of 9mm rounds, I'm fine this being where it is.

Also, if you're going to go to make an energy cake, chances are that you're not the one who purchased that E-sword in the first place. 500 credits as a consolation prize feels appropriate.